### PR TITLE
Add gravatar to csp.

### PR DESCRIPTION
**Pull request by akatsoulas**
@johngian this is relevant to pr #159  . Can you please take a look?

### CODE REVIEW

File: feedthefox/settings.py [hunk 1]
```diff
@@ -255,8 +255,8 @@
 
 CSP_IMG_SRC = (
     "'self'",
-    'https://mozillians.org',
-    'https://mozillians.allizom.org',
+    '*.gravatar.com',
```
Comment (johngian) [inline feedthefox/settings.py]: I would prefer to only allow `https://` scheme.

**Comment by akatsoulas** [comment]
@johngian done

**Comment by johngian** [comment]
Aarrrrr+

**Comment by akatsoulas** [comment]
:beers:

**Pull request merged**